app/population/view/daily.njk

```diff
@@ -0,0 +1,114 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
```

Comment:
From the screenshots it looks like the numbers are a little disconnected from this heading visually. It might be nice to see if you can use some [spacing helper classes](https://design-system.service.gov.uk/styles/spacing/) to create a tighter visual link.

Reply:
Adjusted the spacing of the statsEntry heading